README.md

```diff
@@ -3,7 +3,7 @@
 Generate an http4s service on the blaze backend with Circe.
 
 1. [Install sbt](http://www.scala-sbt.org/0.13/docs/Setup.html)
-2. sbt -sbt-version 0.13.13 new http4s/http4s.g8
+2. sbt -sbt-version 0.13.16 new http4s/http4s.g8
```

Comment:
Yes. As it's what I now was using for the `g8Test` command. I don't see a reason we couldn't target 1.0 in the install command. Unless that somehow interacts with the plugin in which case the 0.9.0 isn't yet 1.0 compatible.

1.0 is going to be a messy transition